docs/02-quick-start/01-installation/02-kubernetes.md

```diff
@@ -1,7 +1,107 @@
 ---
 sidebar_label: Kubernetes
 ---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
 
 # Try Ozone With Kubernetes
 
-**TODO:** File a subtask under [HDDS-9856](https://issues.apache.org/jira/browse/HDDS-9856) and complete this page or section.
+## Minikube
+
+**Requirements:**
+
+* Working minikube setup
+* kubectl
+
+`kubernetes/examples` folder of the Ozone distribution contains Kubernetes deployment resource files for multiple use cases. By default the Kubernetes resource files are configured to use *apache/ozone* image from the Docker Hub.
+
+To deploy it to minikube, use the minikube configuration set:
+
+``` bash
+cd kubernetes/examples/minikube
+kubectl apply -k .
+```
+
+And you can check the results with
+
+``` bash
+kubectl get pod
+```
+
+Note: the `kubernetes/examples/minikube` resource set is optimized for minikube usage:
+
+* You can have multiple Datanodes even if you have only one host (in a real production cluster usually you need one Datanode per physical host)
+* The services are published with node port
+
+### Access the services
+
+Now you can access any of the services. For each web endpoint an additional NodeType service is defined in the minikube k8s resource set. NodeType services are available via a generated port of any of the host nodes:
+
+``` bash
+kubectl get svc
+NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
+datanode     ClusterIP   None            <none>        <none>           27s
+kubernetes   ClusterIP   10.96.0.1       <none>        443/TCP          118m
+om           ClusterIP   None            <none>        9874/TCP         27s
+om-public    NodePort    10.108.48.148   <none>        9874:32649/TCP   27s
+s3g          ClusterIP   None            <none>        9878/TCP         27s
+s3g-public   NodePort    10.97.133.137   <none>        9878:31880/TCP   27s
+scm          ClusterIP   None            <none>        9876/TCP         27s
+scm-public   NodePort    10.105.231.28   <none>        9876:32171/TCP   27s
+```
+
+Minikube contains a convenience command to access any of the NodePort services:
+
+``` bash
+minikube service s3g-public
+Opening kubernetes service default/s3g-public in default browser...
```

Comment:
I haven't tried this, but what does this display? Is it the s3g web UI?

Reply:
Yes, it will provide a URL to access a website.
![image](https://github.com/user-attachments/assets/a3137146-07e1-40d4-a9bb-3b6baa3fa35d)